flights/berks_ceh_clients_flight.yaml

```diff
@@ -38,49 +36,49 @@ entityDefinitions:
         type: "nc.PersonBirthDate"
         column: "DOB"
         transforms:
-        - !<transforms.DateTransform>
-          pattern: ["MM/dd/yy","M/dd/yy","MM/d/yy","M/d/yy"]
-      nc.PersonRace:
+        - !<transforms.DateTimeAsDateTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+      nc.PersonRace1:
```

Comment:
Also, you never need the parseInt transform if the end product will not be an integer.